library/artifactory_repo.py

```diff
@@ -321,26 +326,31 @@
 
 VALID_PACKAGETYPES = ["bower",
                       "chef",
-                      "cocoapads",
+                      "cocoapods",
                       "composer",
                       "conan",
+                      "conda",
+                      "cran",
                       "debian",
                       "docker",
                       "gems",
                       "generic",
                       "gitlfs",
+                      "go",
                       "gradle",
                       "helm",
                       "ivy",
                       "maven",
                       "npm",
                       "nuget",
                       "opkg",
+                      "p2",
                       "puppet",
                       "pypi",
                       "rpm",
                       "sbt",
-                      "vagrant"]
+                      "vagrant",
+                      "vcs"]
```

Comment:
Can you move the `]` to a new line
```suggestion
                      "vcs"
    ]
```